lib/reporters/html/partials/aggregations.hbs

```diff
@@ -0,0 +1,56 @@
+{{#if folder}}
```

Comment:
If the html report template is separated into 2 ".hbs" files, is it possible to customize both the "template-default.hbs" and "aggregations.hbs", and define the templates used in command line call (e.g. --reporter-html-template <path>)?